misago/legal/models.py

```diff
@@ -1 +1,127 @@
 # Empty models file that triggers migrations for this app
+from django.db import models
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+import datetime
+
+
+class NoActiveLegalAgreement(ValidationError):
+    pass
+
+
+class BaseModel(models.Model):
```

Comment:
Don't cut number of lines like that. You are saving two lines per model, but you are also adding great cognitive cost to working with models.